plugins/catalog-react/report-alpha.api.md

```diff
@@ -323,6 +324,102 @@ export const EntityContentBlueprint: ExtensionBlueprint<{
   };
 }>;
 
+// @alpha (undocumented)
+export const EntityHeaderBlueprint: ExtensionBlueprint<{
+  kind: 'entity-header';
+  name: undefined;
+  params:
+    | {
+        defaultFilter?: string | ((entity: Entity) => boolean) | undefined;
+        loader: () => Promise<JSX.Element>;
+      }
+    | {
+        defaultFilter?: string | ((entity: Entity) => boolean) | undefined;
+        title?:
+          | ReactNode
+          | {
+              actions: ReactNode[];
+            };
+        subtitle?: ReactNode;
+      };
```

Comment:
Oh this is an interesting design, will have to ponder it a bit. It certainly slims down the API surface a bit and reduces the complexity of the tree, but on the other hand it limits the ability to provide extensions from plugins that add elements to the header.

Reply:
Let's discuss it 🙂